packages/ui-extensions/src/surfaces/admin/components/Thumbnail/examples/default.html

```diff
@@ -0,0 +1,4 @@
+<s-thumbnail
+  alt="Thumbnail"
```

Comment:
We should lead by example and actually follow our guidelines for the `alt` attribute. 
```suggestion
  alt="White sneakers"
```